site/content/Reference/_index.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "Reference"
+linkTitle: "Reference"
+weight: 3
+description: >
+    Usage and command line flags reference.
+---
+## Introduction
+`vhs` is designed for flexibility and operates on the concept of a data flow that originates with a [source](#sources)
+and terminates with one or more [sinks](#sinks). Sources may capture network data, read data from files, etc. Sinks may
+write data to cloud or local storage, standard output, or send data to other destinations. Along the way, data may pass
+through a series of input [modifiers](#input-modifiers) and [formats](#input-formats) and output 
+[modifiers](#output-modifiers) and [formats](#output-formats) that transform the data. For more information on the 
+technical implementation of `vhs`, see the [architecture overview](/vhs/architecture/).
+
+## Example `vhs` Command
+```./vhs --inputs "tcp|http" --outputs "json|stdout" --address 0.0.0.0:8080 --capture-response```
+
+This command captures two-way TCP data from the network on address `0.0.0.0` and port `8080`, extracts HTTP requests
+and responses from the TCP data, formats them as JSON, and prints them to the standard output.
+
+## Specifying Inputs and Outputs
+The core command line flags for `vhs` are focussed on defining the data flow that `vhs` will use for a recording/replay
+session. Inputs and outputs are specified in terms of a simple domain specific language that will be detailed in
+the following sections.
+
+### Inputs
+```--input "<source|modifier(s)|format>"```
+
+Inputs are specified in a pipe-delimited (`|`), double-quoted string following the `--input` command line flag. 
+The first element in the string must specify a [source](#sources), followed by a pipe character, followed by zero or
+more [modifiers](#input-modifiers) separated by pipe characters, followed by another pipe character, and ending with an
+input [format](#input-formats) specifier. The specified source originates a data stream that is modified by the 
+specified modifiers and then formatted, or interpreted by the specified input format.
```

Comment:
This feels overly verbose, followed by a pipe character. Followed by another pipe character. :)

Since we've already mentioned the input is pipe delimited, maybe something along the lines of `This must begin with an input [source](#sources), optionally contain [modifiers](#input-modifiers), and end with an [input format](#input-formats).`

Reply:
`"Good|call."`